CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Added
```

Comment:
I think for now I prefer to keep a flat namespace here (i.e. no separation between Added/Fixed/etc) - otherwise I think its a good idea to have an Unreleased section.

Reply:
I'll revise the thing, feel free to Squash commit these changes.